overlord/ifacestate/helpers.go

```diff
@@ -859,6 +859,30 @@ func getHotplugAttrs(task *state.Task) (ifaceName, hotplugKey string, err error)
 	return ifaceName, hotplugKey, err
 }
 
+func obtainHotplugSeq(st *state.State) (int, error) {
```

Comment:
Wondering about this name. It suggests that there are no modifications done behind the scenes, which is not true. Maybe something like `assignNextHotplugSeq(st)`? (keep in mind I'm super bad at naming things :smile: )

Reply:
I agree with Maciek, perhaps `nextHotplugSeq` or `allocHotplugId`